.mage/docs.go

```diff
@@ -47,6 +73,14 @@ func (d Docs) yarn() (func(args ...string) error, error) {
 
 // Deps installs the documentation dependencies.
 func (d Docs) Deps() error {
+	fileUrl := "https://raw.githubusercontent.com/TheThingsNetwork/lorawan-frequency-plans/master/frequency-plans.yml"
```

Comment:
Ideally this is configurable, so that deployments that use a fork can also generate their documentation.